lib/game.rb

```diff
@@ -0,0 +1,193 @@
+require 'pry'
+class Game
+	WIN_COMBINATIONS = [
+											[0,1,2],
+											[3,4,5],
+											[6,7,8],
+											[0,3,6],
+											[1,4,7],
+											[2,5,8],
+											[0,4,8],
+											[6,4,2]
+											]
+	DIVIDER = "===================\n"									
+											
+	attr_accessor :player_1, :player_2, :board, :war_mode, :war_stats
+	attr_reader :the_winner, :complete
+	
+	def initialize(player_1=Human.new("X"), player_2=Human.new("O"), board=Board.new)
+		@player_1 = player_1
+		@player_2 = player_2
+		@board = board
+		@complete = false
+		@war_mode = false
+		@war_stats = {
+			:num_games => 0,
+			:x_wins => 0,
+			:o_wins => 0,
+			:draws => 0
+		}
+	end
+	
+	def current_player
+		board.turn_count.even? ? player_1 : player_2
+	end
+	
+	def over?
+		won? ? true : draw?
```

Comment:
Typically, a ternary operator returning a boolean literal is unnecessary. Here, you could simply say
```
def over?
  won? || draw?
end
```
The or operator `||` is short-circuited, meaning it will return true as soon as one of the expressions returns true.